src/js/components/FileInput/FileInput.js

```diff
@@ -55,6 +55,8 @@ const Label = styled(Text)`
     props.theme.fileInput &&
     props.theme.fileInput.label &&
     props.theme.fileInput.label.extend};
+  cursor: ${(props) =>
```

Comment:
Similar comment to above, but this should appear before the extend. Also, instead of adding `default` as a fallback, could we simplify the code in all instances as:

```
props.disabled && props.theme.global.controls.disabled.cursor ? `cursor: ${props.theme.global.controls.disabled.cursor};` : '';
```

Reply:
This should simplify the CSS to not add cursor: default in as many places.